src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -222,15 +223,15 @@ public static String composeRpcMethodHeaderSampleCode(
     List<Statement> bodyStatements = new ArrayList<>();
     if (method.isPaged()) {
       bodyStatements.addAll(
-          composeUnaryPagedRpcMethodSampleCodeBodyStatements(
+          composeUnaryPagedRpcMethodBodyStatements(
```

Comment:
I think Paged can only be Unary, in which case we can remove "Unary" from the name. Could you please confirm?

Reply:
Sure.